commands/core/init.drush.inc

```diff
@@ -0,0 +1,115 @@
+<?php
+
+/**
+ * @file
+ *   Set up local Drush configuration.
+ */
+
+/**
+ * Implementation of hook_drush_command().
+ *
+ * @return
+ *   An associative array describing your command(s).
+ */
+function init_drush_command() {
+  $items['core-init'] = array(
+    'description' => 'Initialize local Drush configuration.',
+    'aliases' => array('init'),
+    'bootstrap' => DRUSH_BOOTSTRAP_NONE,
```

Comment:
Perhaps provide an example of running this on a remote alias (it would work OK?)